Sources/HiPMobile/HiPMobile.csproj

```diff
@@ -274,6 +279,15 @@
     <Reference Include="System.Net.Http.Primitives, Version=4.2.29.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
       <HintPath>..\..\packages\Microsoft.Net.Http.2.2.29\lib\portable-net45+win8\System.Net.Http.Primitives.dll</HintPath>
     </Reference>
+    <Reference Include="Xamarin.Forms.Core, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
`HipMobile` should not have dependencies on UI layer libraries. For what exactly are these three new references needed in this particular project?

Reply:
Removed all Xamarin references from the HiPMobile project file